src/engine/resources.ts

```diff
@@ -784,6 +787,46 @@ export function forceNCPossible(): boolean {
   return forceNCSources.find((s) => s.available()) !== undefined;
 }
 
+type ForceNCSource = {
+  available: () => boolean;
+  do: () => void;
+}
+
+const tuba = $item`Apriling band tuba`;
+
+export const noncombatForceNCSources: ForceNCSource[] = [
+  {
+    available: () => (AprilingBandHelmet.canJoinSection() || have(tuba)) && tuba.dailyusesleft > 0,
+    do: () => AprilingBandHelmet.play(tuba, true),
+  },
+  {
+    available: () => CinchoDeMayo.totalAvailableCinch() >= 60,
+    do: () => useSkill($skill`Cincho: Fiesta Exit`)
+  },
+];
+
+export function tryForceNC(): boolean {
+  if (get("noncombatForcerActive")) return true;
+  noncombatForceNCSources.find((source) => source.available())?.do();
+  return get("noncombatForcerActive");
+}
+
+export function tryPlayApriling(modifier: string): void {
```

Comment:
As long as we don't run out of Apriling plays in a given run this seems fine. If we are bouncing back and forth too much we could integrate this into the priority system as well. Mainly we want to avoid getting stuck on -combat/+combat when we want the other.

Reply:
You can change them every 11 turns, which is pretty frequent. There's a better way to do this, but given that +com and -com are largely grouped, this is probably fine.

An alternative approach may be to only use the food/booze and +com - switching to food/booze when we're done +com sections.